app-header/app-header.html

```diff
@@ -95,6 +95,8 @@
       position: relative;
       display: block;
       -webkit-transform: translateZ(0);
```

Comment:
Is there supposed to be the unprefixed version of `transform` as well? There should also be a `transition-property: -webkit-transform`.